geminidr/core/parameters_preprocess.py

```diff
@@ -53,6 +53,20 @@ class dilateObjectMaskConfig(config.Config):
     dilation = config.RangeField("Dilation radius (pixels)", float, 1., min=0)
     repeat = config.Field("Allow dilation of already-dilated image?", bool, False)
 
+
+class fixPixelsConfig(config.Config):
+    suffix = config.Field("Filename suffix", dtype=str,
+                          default="_pixelsFixed", optional=True)
+    regions = config.Field("Regions to fix", dtype=str, optional=True)
```

Comment:
I think that it would be good to have an example of the format expected for the string defining regions to fix.

Reply:
I'm not sure if putting a longer multi-line string here is a good idea or not ? It seems to be managed correctly by `showpars` does it look good to you ?
```
========================================
 Name			Current setting

suffix               '_pixelsFixed'       Filename suffix
regions              None                 Regions to fix.

Regions must be specified either as a string, separated by semi-colons,
with the ``regions`` parameter, or with a file (``regions_file``), one
region per line.

Regions strings must be a comma-separated list of colon-separated pixel
ranges, one per axis, in 1-indexed Cartesian pixel co-ordinates,
inclusive of the upper limit. Axes are specified in Fortran order
(reverse of the Python order). For example, "450,521;430:437,513:533"
would specify a single pixel (line 521, column 450) and a vertical
region (lines 513 to 533 included, columns 430 to 437).
    
regions_file         None                 Path to a file containing the regions to fix
axis                 None                 Axis over which the interpolation is done
use_local_median     False                Use a local median filter for single pixels?
```

Reply:
No, that doesn't help.  It adds clutter.  Just 'eg. "450,521;430:437,513:533"'   But even then, I don't understand what this means.

450,521...  Is that a pixel *range*?   Since when a range gets expressed with a comma?   Or is that point?

Also, if the coordinates are 1-indexed Cartesian, with Fortran order, why does the "axis" parameter expects "0" as "first axis"?   First axis in Python sense, which then does not match the coordinates in "regions".    That doesn't look good to me.

Reply:
> 450,521... Is that a pixel range? Since when a range gets expressed with a comma? Or is that point?

It's a single pixel. It's mentioned in the example, but I will also update the previous sentence to "colon-separated pixel
_coordinates_ or ranges".

> Also, if the coordinates are 1-indexed Cartesian, with Fortran order, why does the "axis" parameter expects "0" as "first axis"? First axis in Python sense, which then does not match the coordinates in "regions". That doesn't look good to me.

Right... well for regions parsing I reuse `cartesian_regions_to_slices` for which the choice was 1-indexed Fortran order. But for `axis` I just did the natural Python thing... As a Python user I would never think about specifying an axis in Fortran order, not sure why this choice was made for `cartesian_regions_to_slices` ?